examples/models/unet.py

```diff
@@ -0,0 +1,105 @@
+import torch.nn as nn
+from models.modules.blocks import (SparseConvBlock, SparseConvTransposeBlock,
+                                   SparseResBlock)
+
+import torchsparse
+import torchsparse.nn as spnn
+from torchsparse.tensor import SparseTensor
+
+__all__ = ['SparseResUNet18']
+
+
+class SparseResUNet18(nn.Module):
+
+    def __init__(self,
+                 in_channels: int = 4,
+                 channel_sizes: list = None,
+                 width_multiplier: float = 1.0):
+        super().__init__()
+
+        cs = channel_sizes if channel_sizes is not None else [
+            32, 32, 64, 128, 256, 256, 128, 96, 96
+        ]
+        cs = [int(width_multiplier * x) for x in cs]
+
+        self.stem = nn.Sequential(
+            spnn.Conv3d(in_channels, cs[0], kernel_size=3, stride=1),
+            spnn.BatchNorm(cs[0]),
+            spnn.ReLU(True),
+            spnn.Conv3d(cs[0], cs[0], kernel_size=3, stride=1),
+            spnn.BatchNorm(cs[0]),
+            spnn.ReLU(True),
+        )
+
+        self.downs = []
```

Comment:
You can directly initialize this as `nn.ModuleList()` which also supports `append` just as normal list.